lib/atom-alias-command.coffee

```diff
@@ -1,10 +1,15 @@
 {CompositeDisposable} = require 'atom'
 
+typeIsArray = Array.isArray || ( value ) -> return {}.toString.call( value ) is '[object Array]'
+
 module.exports = aliasCommand = (as, {orig, scope}) ->
   scope ?= 'atom-workspace'
   aliasCommand.subscriptions.add(
     atom.commands.add scope, as, (event) ->
-      atom.commands.dispatch event.target, orig
+      if typeIsArray orig
+        atom.commands.dispatch event.target, o for o in orig
```

Comment:
I couldn't find great info in the Atom docs about the queuing behavior of commands.  For example if I `orig: dispatch('core:save')` then `dispatch('core:close')`, does Atom guarantee that save will complete before we close the file? Perhaps the two commands can be executed in any order.  

We don't need to answer this now if it's working for you.  Just something to keep in mind if you notice any flakiness saving.